augur/io/metadata.py

```diff
@@ -138,7 +138,7 @@ def read_table_to_dict(table, duplicate_reporting=DataErrorMethod.ERROR_FIRST, i
     duplicate_ids = set()
     with open_file(table) as handle:
         # Get sample to determine delimiter
-        table_sample = handle.read(1024)
+        table_sample = handle.readline()
```

Comment:
Oh and also, I don't think it's necessary since we don't limit reads later in this function, but we could preserve the 1KiB read limit with:

```python
… = handle.readline(1024)
```

Reply:
Yeah, I didn't think it would be necessary to add a size limit here since we read full records later anyways.